cmd/addrconv/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/schancel/cashaddr-converter/address"
+	"github.com/schancel/cashaddr-converter/cashaddress"
+	"github.com/schancel/cashaddr-converter/legacy"
+)
+
+type Output struct {
+	CashAddr string `json:"cashaddr,omitempty"`
+	Legacy   string `json:"legacy,omitempty"`
+	Copay    string `json:"copay,omitempty"`
+	Hash     string `json:"hash,omitempty"`
+}
+
```

Comment:
Consider changing the help message using flag.Usage (see https://golang.org/pkg/flag/#pkg-variables)